src/app/campaign-list/list/campaign-list.component.ts

```diff
@@ -0,0 +1,138 @@
+import { Component, OnChanges, ChangeDetectionStrategy, Input } from '@angular/core';
+import { Router } from '@angular/router';
+import { Observable } from 'rxjs';
+import { Campaign, CampaignListService, Facets, CampaignParams } from '../campaign-list.service';
+
+@Component({
+  selector: 'grove-campaign-list',
+  template: `
+    <grove-campaign-filter
+      [params]="params" [facets]="facets" (campaignListParams)="routeToParams($event)">
+    </grove-campaign-filter>
+    <h2>Campaigns</h2>
+    <ul>
+      <li *ngFor="let campaign of campaigns$ | async">
+        <grove-campaign-card [campaign]="campaign"></grove-campaign-card>
+      </li>
+      <ng-container *ngIf="loading$ | async as loading">
```

Comment:
Slick way of doing loading indicators!

Reply:
They arent really showing now that I added the zoom, but it did seem cool at the time! I was trying to do some kind of skeleton interface while loading, but maybe we'll decide something else for loading indication once the design settles in.